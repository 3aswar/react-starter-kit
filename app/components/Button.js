import styled from 'styled-components';

export default styled.button`
  background: transparent;
  border: 2px solid palevioletred;
  border-radius: 3px;
  color: palevioletred;
  margin: 0 1em;
  padding: 0.25em 1em;
`;
