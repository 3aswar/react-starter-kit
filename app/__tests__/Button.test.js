import React from 'react';
import { render } from 'react-testing-library';
import Button from '../components/Button';

it('it works', () => {
  const { container } = render(<Button />);
  expect(container.firstChild).toMatchSnapshot();
});
