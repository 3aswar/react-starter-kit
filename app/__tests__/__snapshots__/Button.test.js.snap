// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  background: transparent;
  border: 2px solid palevioletred;
  border-radius: 3px;
  color: palevioletred;
  margin: 0 1em;
  padding: 0.25em 1em;
}

<button
  class="c0"
/>
`;
